README.md

```diff
@@ -76,7 +76,7 @@ def play(self):
 to:
 
 ```python
-def play(self):
+def play(self, debug):
```

Comment:
By default, the `game.play()` function within `app.py` doesn't contain any other parameter; so after we changed the `Game.py`, if we still want to run `app.py`, a 'Game.play() missing 1 required positional argument: 'debug'' error will be thrown.

A possible solution maybe :
Define the `debug` parameter in the `__init__` function of Game class, with a default value of False.

```python
class Game:
	def __init__(self, player, map, maxEnergy, latency, visuals, cls, debug= False):
		// ...
		self.debug = debug

	def play(self):
		// ...
			if self.debug:
				// ...
```

Reply:
Oh yeah my bad, I actually forgot to add this to the readme. I simply added the `debug` parameter to the `play` method in `game.py` like so: `def play(self, debug)`. 

Both solutions can work I think. The main difference I see is that having debug as parameter in each function call can be verbose and might require changes that are more spread out over files. Up to you!